editor/src/components/navigator/left-pane/pages-pane.tsx

```diff
@@ -128,28 +136,40 @@ export const PagesPane = React.memo((props) => {
     setNavigateTo(null)
   })
 
-  if (remixRoutes.length === 0) {
-    return (
-      <FlexColumn
-        style={{
-          height: '100%',
-          overflowY: 'scroll',
-          whiteSpace: 'normal',
-          padding: 16,
-          paddingTop: 64,
-        }}
-      >
-        <Subdued>
-          No featured routes were found in the project. Please add a 'featuredRoutes' field to the
-          'utopia' field in the package.json file.
-        </Subdued>
-      </FlexColumn>
-    )
-  }
+  const activeRoute = matchResult?.[0].pathname ?? pathname
+  const activeRouteDoesntMatchAnyFavorites = !featuredRoutes.includes(activeRoute)
+
   return (
     <FlexColumn style={{ height: '100%', overflowY: 'scroll' }}>
       <InspectorSectionHeader>
-        <FlexRow style={{ flexGrow: 1 }}>Pages</FlexRow>
+        <FlexRow style={{ flexGrow: 1 }}>Favorites</FlexRow>
+      </InspectorSectionHeader>
+      <FlexColumn style={{ paddingBottom: 24 }}>
+        {featuredRoutes.map((favorite: string) => {
```

Comment:
not for this PR, but we could/should be aggressive and make sure you don't have any duplicates in the favorite routes (for the same Remix path) so we avoid having a bunch of issues with duplicate keys

Reply:
it's a feature to have featured routes showcasing the same remix path with different variables!